### ISSUE TITLE
Accessibility issues - icons.brand.uiowa.edu

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa801#9742]
### Siteimprove flagged issues/potential issues:
- Heading is not descriptive - false positive in my opinion, "All Icons" seems to describe the page fine
- Page language not identified - needs to be fixed - #9733 
- Possible to skip to main content? - manual testing revealed no skip link exists, needs to be fixed - Siteimprove best practice

### Manual testing issues
- No way to close the selected icon modal via keyboard navigation. Update: can close using esc key, but it would still be good to have the close modal button accessible via keyboard navigation - #9735 
- No way to select icon variants in the icon modal via keyboard navigation - #9735
- No way to access color toggle via keyboard navigation - #9736 
- Search box missing field label according to checker - #9738 
- "Download all" button's icon is missing alt text - #9739 
- "Download" options in an icon's modal are missing alt text for the download arrow image as well - #9739 
- nice-to-have possibly, but all color variants of an icon have identical alt text. Additionally, there is no surrounding text labelling the variants, so some kind of differentiation would be nice. - #9739 

### Additional testing notes:
- When using the link text checker, the link and information icons are flagged. I ignored as false positives since "link" and "information" are accurate descriptions of those icons.
- All _text_ passes the color contrast guidelines. The gold line only variant displaying on a white background would fail if it was text, which is another reason I think identifying the variants would be useful.

**Comment by briand44** [comment]
## Grooming
 
**Quadrant:** 

Complex

 **Discussion:**
 - Group these issues together
 - See if we can resolve issues without revamping the icon browser site
 - Skip to main content issue reported as best practice
 - All color variants of an icon have identical alt text is not being flagged, seems like a nice to have

**Proposed solution or definition of done:** 
- Page language set and Siteimprove no longer reporting as issue
- Icon modal should be keyboard navigable, can close out via keyboard, select variant via keyboard, don't have to tab through all icons to access modal via keyboard
- can toggle color on/off via keyboard
- label exists for search box and no longer reported as issue by WAVE/field label checker
- download all icons and download buttons in modal have `role="presentation"` and not flagged by image checker

**Comment by joewhitsitt** [comment]
going to see about vibe coding our way out of this one in order to kick the can a bit.

**Comment by joewhitsitt** [comment]
https://github.com/uiowa/brand-icons/pull/40 was merged
